support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -5,7 +5,7 @@ import type { Tests } from './abtest';
 
 const landingPage = '/??/contribute|thankyou(/.*)?$';
 const usLandingPage = '/us/contribute(/.*)?$';
-const allLandingPagesAndThankyouPages = '/??/contribute|thankyou(/.*)?$';
+const allLandingPagesAndThankyouPages = '/contribute|thankyou(/.*)?$';
```

Comment:
Simplifying this as the `/??` is actually doing nothing

Reply:
Do we still need the `/??` on patterns for other pages? Eg. `landingPage`.

Reply:
yeah that's a good point - I don't think we do! In fact, `landingPage` is actually identical to `allLandPagesAndThankyouPages`! I'll combine those into one.